mlflow/java/client/src/main/java/com/databricks/api/proto/mlflow/DatabricksArtifacts.java

```diff
@@ -2751,32 +2751,68 @@ public ArtifactCredentialInfo parsePartialFrom(
 
```

Comment:
Automatically regenerated to reflect changes in `mlflow/protos/databricks_artifacts.proto` (see https://github.com/mlflow/mlflow/pull/4402/files#r641035500).